app/plugins/Awards/tests/TestCase/Services/RecommendationBestowalStatePolicyServiceTest.php

```
@@ -0,0 +1,41 @@
+<?php
+declare(strict_types=1);
+
+namespace Awards\Test\TestCase\Services;
+
+use Awards\Services\RecommendationBestowalStatePolicyService;
+use Cake\TestSuite\TestCase;
+use RuntimeException;
+
+class RecommendationBestowalStatePolicyServiceTest extends TestCase
```

PHPStan (2.2.7)

[error] Class Awards\Test\TestCase\Services\RecommendationBestowalStatePolicyServiceTest extends unknown class Cake\TestSuite\TestCase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)